src/features/island/buildings/components/building/market/Crops.tsx

```diff
@@ -85,114 +84,85 @@ export const Crops: React.FC = () => {
-    if (cropAmount.greaterThan(1) || cropAmount.eq(0)) return "Sell 1";
+    if (cropAmount.greaterThanOrEqualTo(1) || cropAmount.eq(0)) return "Sell 1";
 
     return `Sell ${cropAmount}`;
   };
 
-  useEffect(() => {
-    setIsPriceBoosted(hasSellBoost(inventory));
-  }, [inventory, state.inventory]);
-
   return (
-    <div className="flex flex-col-reverse sm:flex-row">
-      <div
-        className="w-full sm:w-3/5 h-fit max-h-48 sm:max-h-96 overflow-y-auto scrollable overflow-x-hidden p-1 mt-1 sm:mt-0 sm:mr-1 flex flex-wrap"
-        ref={divRef}
-      >
-        {Object.values(cropsAndFruits)
-          .filter((crop) => !!crop.sellPrice)
-          .map((item) => (
-            <Box
-              isSelected={selected.name === item.name}
-              key={item.name}
-              onClick={() => setSelected(item)}
-              image={ITEM_DETAILS[item.name].image}
-              count={inventory[item.name]}
-              parentDivRef={divRef}
-            />
-          ))}
-      </div>
-      <OuterPanel className="w-full flex-1">
-        <div className="flex flex-col justify-center p-2 pb-0">
-          <div className="flex space-x-2 justify-start mb-1 sm:items-center sm:flex-col-reverse md:space-x-0">
-            <img
-              src={ITEM_DETAILS[selected.name].image}
-              className="w-5 object-contain sm:w-8 sm:my-1"
-              alt={selected.name}
-            />
-            <span>{selected.name}</span>
-          </div>
-          <span className="text-xs sm:text-sm sm:text-center">
-            {selected.description}
-          </span>
-          <div className="border-t border-white w-full my-2 pt-1 flex justify-between sm:flex-col sm:items-center">
-            <div className="flex justify-center space-x-1 items-center sm:justify-center">
-              <img src={token} className="h-4 sm:h-5" />
-              {isPriceBoosted && <img src={lightning} className="h-5 sm:h-6" />}
-              <span className="text-xs text-shadow text-center">
-                {`${displaySellPrice(selected)}`}
-              </span>
-            </div>
-          </div>
-        </div>
-        <div className="flex space-x-1 mb-1 sm:space-x-0 sm:space-y-1 sm:flex-col w-full">
-          <Button
-            disabled={cropAmount.eq(0)}
-            className="text-xxs sm:text-xs"
-            onClick={handleSellOneOrLess}
-          >
-            {sellOneButtonText()}
-          </Button>
-          <Button
-            disabled={cropAmount.lessThan(10)}
-            className="text-xxs sm:text-xs"
-            onClick={handleSellTen}
-          >
-            Sell 10
-          </Button>
-        </div>
-        <Button
-          disabled={noCrop}
-          className="text-xxs sm:text-xs"
-          onClick={openConfirmationModal}
-        >
-          Sell All
-        </Button>
-      </OuterPanel>
+    <>
+      <SplitScreenView
+        divRef={divRef}
+        header={
```

Comment:
I wonder if `header` is the right name for this prop? Whenever I see it I think of `title`. Would it be better as `topPanel` or even `panel`. 🤔

Reply:
Thanks for the suggestion, I renamed it to `panel`.